pageserver/src/http/openapi_spec.yml

```diff
@@ -420,25 +420,6 @@ paths:
           description: Tenant scheduled to load successfully
 
   /v1/tenant/{tenant_id}/synthetic_size:
-    parameters:
-      - name: tenant_id
```

Comment:
This field was called `tenant_id`, but the one in the block below that replaces it called `id`

Reply:
My mistake, I was comparing the tenant_id in the request with the `id` in the response